### ISSUE TITLE
Enable Normal Map Zoom when scroll up/down and mouse pointer hovers markers.

### ISSUE THREAD

**Issue by arnaudgregoire** [sothawo/mapjfx#48]
Currently, when the mouse pointer hover a marker on the map, the normal behavior "scroll down/up to zoom" is disabled. If my mouse pointer hover a marker and i scroll up/down, the current zoom is not changed. I have to change a bit the position of my mouse pointer to make the zoom work again.

I tried to see if a MarkerEvent was fired when i scroll up/down with a mouse pointer on a marker, but it seems that no events are triggered.
```java
        mapView.addEventHandler(MarkerEvent.ANY, event -> {
            System.out.println("marker event fired");
        });
```
i also look for MapViewEvent but same result appear. No events are triggerd.

```java
        mapView.addEventHandler(MapViewEvent.ANY, event -> {
            System.out.println("Map View event fired");
        });
```
It would be nice to allow the same behavior "scroll down/up to zoom" even if the mouse pointer hover a marker.

**Comment by sothawo** [comment]
thanks for pointing that out, won't be before next week that I will have time to fix this.

**Comment by sothawo** [comment]
implemented  in 1.24.0 and 2.5.0
